build/gen_stub.php

```diff
@@ -637,8 +645,11 @@ function parseFunctionLike(
             $param->default->name->toLowerString() === "null" &&
             $type && !$type->isNullable()
         ) {
-            throw new Exception(
-                "Parameter $varName of function $name has null default, but is not nullable");
+            $simpleType = $type->tryToSimpleType();
```

Comment:
I could also imagine adding and using a `$type->isImplicityNullable()` method if this solution feels hacky.